draft-ietf-add-ddr.md

```diff
@@ -91,7 +91,8 @@ Both of these approaches allow clients to confirm that a discovered Encrypted
 Resolver is designated by the originally provisioned resolver. "Designated" in
 this context means that the resolvers are operated by the same entity or
 cooperating entities; for example, the resolvers are accessible on the same
-IP address, or there is a certificate that claims ownership over both resolvers.
+IP address, or there is a certificate that claims ownership over the designated
+resolver IP address (regardless of what IP the designated resolver is on).
```

Comment:
I think the cert covers the original "designating" IP address, not the "designated" IP address.
Since one method uses the name, I think this should either be:

```suggestion
IP address, or there is a certificate that claims ownership over the original
resolver IP address or hostname.
```
or:
```suggestion
IP address, or there is a certificate that claims ownership over the original
designating resolver.
```